external/urlfrontier/src/main/java/com/digitalpebble/stormcrawler/urlfrontier/Spout.java

```diff
@@ -71,25 +71,27 @@ public void open(
             int nodeIndex = context.getThisTaskIndex();
             Collections.sort(addresses);
             address = addresses.get(nodeIndex);
+        } else if (addresses.size() == 1) {
```

Comment:
why not be flexible and let users specify a single address? Won't do any harm

Reply:
OK, thanks. If it isn't working then let's not allow it as per your suggestion. We can always fix it later